components/style/values/computed/mod.rs

```diff
@@ -494,6 +495,24 @@ impl SVGPaint {
 /// <length> | <percentage> | <number>
 pub type LengthOrPercentageOrNumber = Either<Number, LengthOrPercentage>;
 
+#[cfg(feature = "gecko")]
+impl ToAnimatedValue for LengthOrPercentageOrNumber {
```

Comment:
Thanks hiro, 

Ah, I got you now. The Au is i32, so the length can't present value if we specify too small value.

You mean, the following my understanding is right?

specified value = `LengthOrPercentageOrNumber`
animated value = `Number Or Percentage` (converted length to number)
computed value = `LengthOrPercentageOrNumber`